doc/PLUGIN.md

```diff
@@ -0,0 +1,44 @@
+# Plugin API
+
+`Miradord` makes use of external plugins to define and enforce spend
+policies.
+
+## Hooks
+
+### `new_block`
+
+Every block the watchtower is calling the plugins which paths are in the
+configuration file.
```

Comment:
```suggestion
Upon notifying a new block chain tip, if the state of vaults under watch changed, `miradord` will call into every plugin with information about the state updates. It will expect a response containing a list of vaults to re-vault (which may be empty).
```